src/app/work-item/work-item-board/work-item-board.component.scss

```diff
@@ -57,6 +57,9 @@
               justify-content: space-between;
             }
           }//boardcard ended
+          .board-card-selected {
+            border: 3px solid #bee1f4;
+          }
         }//boardlaneWrapper ended
         .lane-header {
           text-align: $textCenter;
```

Comment:
Sorry @mindreeper2420 , i haven't touch that code yet... but you have a valid point. I will talk to @vikram-raj or @sanbornsen .